docs/fast_data/configuration/projection_storer.md

```diff
@@ -0,0 +1,1048 @@
+For debugging purposes, a logger is still provided via internal bindings, but it is recommended to not log
+within cast functions at levels higher than `debug`.
+:::
+
+### Consumer
+
+| Property        | Type     | Required | Default |
+|-----------------|----------|----------|---------|
+| `type`          | `string` | &check;  | `kafka` |
+| `configuration` | `object` | &check;  |         |
+
+Describe which type of consumer and its configuration properties the service should employ to read ingestion messages
+as input events. Currently only Kafka (and platforms adopting Kafka APIs) is supported as consumer.
+
+#### Kafka Configuration
+
+When Kafka is selected as consumer for the Projection Storer service, it is possible to provide most of the Kafka Consumer
+properties that are defined in the [Apache Kafka documentation](https://kafka.apache.org/documentation/#consumerconfigs).
+
+This is an example of consumer configuration when `kafka` is selected as type: 
+
+```json
+"consumer": {
+  "type": "kafka",
+  "configuration": {
+    "client.id": "galaxy.fast-data.DEV.inventory-projection-storer-consumer",
+    "bootstrap.servers": "localhost:9092",
+    "group.id": "galaxy.fast-data.DEV.inventory-projection-storer",
+    "auto.offset.reset": "latest",
+    "max.poll.records": 2000,
+    "max.poll.timeout.ms": 500
+  }
+}
+```
+
+:::info
+The consumer property `max.poll.timeout.ms` is an ad-hoc property, which does not belong to the set of Kafka Consumer properties.
+In fact, it is employed by the service to set the maximum number of milliseconds the consumers waits for a poll operation
+before returning any event (in case `max.poll.records` or `message.max.bytes` are not reached earlier than the configured timeout).
+:::
+
+:::note
+The following Kafka Consumer properties cannot be customized by the user, since are managed by the service:
+- `enable.auto.commit`
+- `allow.auto.create.topics`
+- `key.deserializer`
+- `value.deserializer`
+:::
+
+### Producer
+
+| Property        | Type     | Required | Default |
+|-----------------|----------|----------|---------|
+| `type`          | `string` | &check;  | `kafka` |
+| `configuration` | `object` | &check;  |         |
+
+Describe which type of consumer and its configuration properties the service should employ to trigger change 
+as input events. Currently only Kafka (and platforms adopting Kafka APIs) is supported as consumer.
+
+#### Kafka Configuration
+
+When Kafka is selected as producer for the Projection Storer service, it is possible to provide most of the Kafka Producer
+properties that are defined in the [Apache Kafka documentation](https://kafka.apache.org/documentation/#producerconfigs).
+
+This is an example of producer configuration when `kafka` is selected as type:
+
+```json
+"producer": {
+  "type": "kafka",
+  "configuration": {
+    "client.id": "galaxy.fast-data.DEV.inventory-projection-storer-producer",
+    "bootstrap.servers": "localhost:9092"
+  }
+}
+```
+
+:::note
```

Comment:
```suggestion
:::caution
```

Reply:
I would leave as a `note`, since it is something the user need to be aware, but it does not cause any issue when configuring the service